pythreepio/pythreepio/utils.py

```diff
@@ -1,11 +1,15 @@
+#importlib.resources is imported  to provide access to resources within packages
 try:
     import importlib.resources as pkg_resources
+#for easier accessibility  importlib_resources can be refered to as pkg_resources
```

Comment:
This except statement allows us to use `importlib.resources` in python3.6 and lower. Could you update this comment to say that? :)